### PR TITLE
Feat: Added Signup page

### PR DESCRIPTION
**Pull request by deepak29-git**
Added Signup Page with Navbar

https://user-images.githubusercontent.com/88363801/153502309-8900462f-e490-465a-a3e0-54e9e9393dbf.mp4

### CODE REVIEW

**Review by ajay-mandaviya — APPROVED** [review]
Great

File: Cart-management/cart.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    justify-content: space-between;    
+}
+
+
+.placeholder-btn{
+    display:flex;
+    justify-content: center;
+    background-color:var(--primary-color);
+    color:var(--white-color);
+    padding:0.8rem 0;
```
Comment (anshikaa12) [inline Cart-management/cart.css:24]: you should use px for<16 px

**Review by anshikaa12 — APPROVED** [review]

**Review by spraveenofficial — APPROVED** [review]
Looks good to merge.

File: Cart-management/cart.css (hunk 1 above)
Comment (deepak29-git) [reply Cart-management/cart.css:24]: done

**Pull request merged**